openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -98,6 +97,8 @@ class AccountView(APIView):
             * year_of_birth: The year the user was born, as an integer, or null.
             * account_privacy: The user's setting for sharing her personal
               profile. Possible values are "all_users" or "private".
+            * has_accomplishments: Signals whether badges are enabled on the
```

Comment:
Hmm.. The way I had interpreted the mobile UI requirements was that the "accomplishments" tab would appear only if badging was enabled, permissions allowed, *and* badges actually existed.  That is what we had written in our ticket: https://openedx.atlassian.net/browse/MA-1670.  However, this may have been a misinterpretation of the requirements.  And, if it over-complicates the code, it may not be worth it.  
FYI @explorerleslie.  Please note that we will show the accomplishments tab even if the user has no badges.  In the long run, this may be good, since we can entice the user to earn badges on an empty screen.